byzcoin/README.md

```diff
@@ -104,6 +104,98 @@ The design of the view-change is similar to the view-change protocol in PBFT
  correct. This technique enables nodes to synchronise and replay blocks to
  compute the most up-to-date leader.
 
+## Creation of Blocks
+
+This is the path a transaction takes from the client to the block:
+
+1. A client creates a `ClientTransaction` and sends it to one or more nodes.
+ Sending it to more than one node increases the probability that at least
+ one node correctly forwards the `ClientTransaction`.
+
+2. The nodes send the `ClientTransaction` to the leader.
+
+2.a If the leader is not responding, they send the `ClientTransaction` to
+ the other nodes, indicating that the node failed to send the transaction.
+ Every node that fails to send such a transaction to the leader will start
+ requesting a viewchange.
```

Comment:
previously view change was written as 2 words, sometimes hyphened (line 91, 98). It would be nice to be consistent so that it's easier to navigate the file.

Reply:
In https://arxiv.org/pdf/1602.06997.pdf it's mostly `view-change`, and only some `view change`, so I'm taking the one with the hyphen.